src/components/checkBoxSelection/checkBoxSelection.scss

```diff
@@ -29,24 +20,59 @@
 
 .file {
   display: flex;
-  margin-top: 20px;
+      padding: 2%;
 }
 
 .hide-file-list {
-  margin-top: 20px;
   display: none;
 }
-
 .card-settings {
-  height: 600px;
+  height: 350px;
   overflow-y: scroll;
+  background-color:#b5cfe1 ;
+  width: 600px;
+}
+
+.home-display-class {
+  display: flex;
+}
+.choose {
+  width: 50%;
+}
+.display {
+  width: 50%;
+}
+
+.optimize-button{
+  text-align: center;
+  display: inline-block;
+  justify-items: center;
+  width:100px;
+  height: 70px;
+  border-radius: 25px;
   background-color: #b5cfe1;
-  width: 350px;
-  padding-left: 20px;
-  box-shadow: 10px 10px 55px 0px black;
+  align-items: center;
+}
+
+// .download-Button{
+//     padding-left: 50%;
+//     padding-top: 3%;
+// }
+.download-Button {
```

Comment:
ghost code

Reply:
removed.

Reply:
doesnt look like it, please push if you do have any changes

Reply:
Can you double check again, I have been pushing more frequently after your prev comment.

Reply:
https://www.w3schools.com/jsref/jsref_obj_array.asp this would help you find available methods on arrays, look for methods that say "returns a new array and does not change the original one"

Reply:
This was found by googleing :
"immutable remove from array javascript"
https://stackoverflow.com/questions/47023975/what-is-the-cleanest-way-to-remove-an-element-from-an-immutable-array-in-js?noredirect=1&lq=1

Reply:
I had an idea and wanted to know how you would like it so I do not do this over and over again,thanks

Reply:
cool